app/src/main/java/org/stepic/droid/core/StepikLogoutManager.kt

```diff
@@ -48,10 +52,11 @@ class StepikLogoutManager
     private fun removeCookiesCompat() {
         CookieManager.getInstance()
         if (Build.VERSION.SDK_INT < 21) {
+            @Suppress("DEPRECATION")
             CookieManager.getInstance().removeAllCookie()
         } else {
             mainHandler.post {
-                CookieManager.getInstance().removeAllCookies() {}
+                CookieManager.getInstance().removeAllCookies {}
```

Comment:
I think better solution is `removeAllCookies(null)`, because it is more clear and extra object won't be created.

From documentation:
>      * You can pass {@code null} as the callback if you don't need to know when the operation
>      * completes or whether any cookies were removed, and in this case it is safe to call the
>      * method from a thread without a Looper.